flang/lib/Semantics/check-io.cpp

```diff
@@ -9,7 +9,9 @@
 #include "check-io.h"
 #include "definable.h"
 #include "flang/Common/format.h"
+#include "flang/Common/indirection.h"
```

Comment:
FYI, The [llvm policy about headers](https://llvm.org/docs/CodingStandards.html#include-as-little-as-possible) says that it is OK to include indirectly and I am fairly sure these two headers are included indirectly already via the tools.h. I am fine with the explicit include (so is the policy), but just know that you do not need to bother adding explicit includes.